lib/manageiq/appliance_console/internal_database_configuration.rb

```diff
@@ -8,8 +8,8 @@ module ApplianceConsole
   class InternalDatabaseConfiguration < DatabaseConfiguration
     attr_accessor :disk, :run_as_evm_server
 
-    DEDICATED_DB_SHARED_BUFFERS = "'1GB'".freeze
-    SHARED_DB_SHARED_BUFFERS = "'128MB'".freeze
+    DEDICATED_DB_SHARED_BUFFERS = "'3GB'".freeze
+    SHARED_DB_SHARED_BUFFERS = "'2GB'".freeze
```

Comment:
I think these values are a bit high.

ManageIQ appliances ship with 6GB of RAM and by default run both the database and the application.

So I'm thinking 500MB for shared and 2GB for dedicated?

Reply:
You're correct, that is high if MIQ ships with 6GB of RAM. Would it make sense to have CloudForms and ManageIQ align with the memory footprint/appliance specs by bumping it to a 4 x 12GB appliance?